internal/ir/builder.go

```diff
@@ -479,3 +521,30 @@ func (b *builder) buildFuncParameter(fn *Function, expr ast.Expr) *Parameter {
 		return nil
 	}
 }
+
+// addEdge adds a control-flow graph edge from from to to.
```

Comment:
Actually not, I'm meaning from to <to> where <to> is the parameter value. I think that this sounds a bit confusing, any recommendation?

Reply:
Maybe just specify that are referring to the parameters, something like this: `adds a control-flow graph edge from parameters from to.`

Reply:
Since this function is very simple, what do you think about rename the parameter to `f`and `t` and change the doc to `// addEdge adds a control-flow graph edge from f to .t`?

Reply:
Sounds good